src/components/TableMatrix/TableHeader/TableHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from '../TableHeader/TableHeader.module.scss';
+
+const topCells = (value: string | number) => {
+  return (
+    <th className={styles.headerItem} key={value}>
+      {value}
+    </th>
+  );
+};
+
+const TopUnit = (columns: number) => {
+  const banner = [topCells('№')];
+  for (let i = 0; i < columns; i++) {
```

Comment:
it is better to use **_forEach_** method instead of _**for**_ because _**forEach**_ has a more understandable and simpler syntax